### DISCUSSION TITLE
Welcome to Nightingale Discussions!

### DISCUSSION THREAD

**Discussion by RAJANAGORI** [RAJANAGORI/Nightingale#11] [Ideas]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  For more information to know more about the Nightingale before start any conversation, you can hit here - [Wiki](https://github.com/RAJANAGORI/Nightingale/wiki)

**Comment by m2sup3rn0va** [comment]
Hey Guys

This is a great initiative and thank you for making this publicly available.
Even I would love to contribute because I already have a similar project in my GitHub as well..

[**M2Sup3rN0va - Dr0P**](https://github.com/m2sup3rn0va/Dr0P)

This is mainly focused on Mobile (Android/iOS) PT including SAST and DAST and parts of Web PT like initial recon and all.
I hope my work will help this project to add more on Mobile PT side.

Also, please have a look at below as well. It's an automation script to install all the tools required for Mobile (Android/iOS) PT.

[**M2Sup3rN0va - SiAAA**](https://m2sup3rn0va.github.io/SiAAA/)

Please do let me know if I can be of any assistance.

M2Sup3rN0va

**Reply by RAJANAGORI** [reply]
Awesome stuff !!
Really Appreciated that someone has the great solution in the scope of Mobile Pentesting.

It would be great if you will be a part of this tool and contribute your ideas for Mobile Pentesting.

Looking forward to your PR 

Regards,
Raja

**Reply by m2sup3rn0va** [reply]
Hey 

Thank you and appreciate your response on this.
Definitely will add up to the tool.

Thank you again.

Regards
M2Sup3rN0va

**Comment by darsh4884** [comment]
Hey,
I'm not sure, if i contribute in any way as it's not my niche. But i can suggest one thing.
I see this tool doesn't have a website yet, so how about we create one using ReactJS and maybe we can add some analytics so as to know who accessed the website from where and in what section they're interested in. Azure or GCP can help us achieve that.

**Reply by RAJANAGORI** [reply]
Cool Idea, yes we can
Need some logic which can render quickly and transfer the content of the server over the webpage.

That will be an optional for those users who love GUI over CLI haha

Looking forward to your PR.

Regards,
Raja

**Reply by RAJANAGORI** [reply]
Hey @darsh4884 , just a heads up on your request - This tool has a website and a working product dashbard. You can find both on the mentioned URL respectivbely 
https://nightingale-security.com/ and https://dashboard.nightingale-security.com/login

**Comment by dhirajberi** [comment]
Hey Guys, I'd like to offer some suggestions.
1. Implement a user interface (UI) for the tool.
2. Monitoring system for the tool.

**Reply by RAJANAGORI** [reply]
Thank you so much @dhirajberi for the suggestion.
I have created an issue to track your suggestion as I started working on it 😉 
#46

**Reply by RAJANAGORI** [reply]
Hey @dhirajberi  
Here is the idea behind implementing the Apache Guacamole https://github.com/RAJANAGORI/Nightingale/issues/58 you can find the more information here.

**Reply by RAJANAGORI** [reply]
@dhirajberi Hey, just a heads up on your request - This tool has a website and a working product dashbard. You can find both on the mentioned URL respectivbely
https://nightingale-security.com/ and https://dashboard.nightingale-security.com/login
